### DISCUSSION TITLE
Only "member" of team, need to become maintainer

### DISCUSSION THREAD

**Discussion by bsuwal** [community/community#46781] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 

I am listed as a "member" of a team that has only one person (me). I would like to be the maintainer (i dont know how i came to this predicament?) Is there some kind of process for this??

**Comment by SilasRodrigues19** [comment]
Hey @bsuwal 

Try these steps in the link below:

https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member

**Comment by bsuwal** [comment]
Hi, thanks!

These steps can be followed by only the team maintainer. From the link you just sent me:

"Organization owners can promote team members to team maintainers."

My issue is that i am not the org owner. I am only a member. But I am the only person in my org (it doesn't have anyone with the "maintainer" role)

**Comment by bsuwal** [comment]
bumping this up!

**Comment by queenofcorgis** [comment]
Hey @bsuwal - thanks for posting this. I've been reading through our docs to see if there was anything in there to help, but I think this may be a case for [support](https://support.github.com/). 

Also, you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by bsuwal** [reply]
thanks @queenofcorgis !

**Reply by queenofcorgis** [reply]
You're welcome - let me know how you make out!

**Reply by ShahafBenYakir** [reply]
@bsuwal Did you manage to sort this one out? I am having the same issue
